app/Http/Controllers/NewCatalogController.php

```diff
@@ -14,7 +15,19 @@ class NewCatalogController extends Controller
      */
     public function index()
     {
-        $authors = Authority::latest()->take(10)->get();
-        return view('frontend.catalog-new.index-new',['authors' => $authors]);
+        $authors = Authority::latest()
+            ->take(10)
+            ->get();
+
```

Comment:
If you can delete this please I will be grateful :)

```suggestion
        $authors = Authority::latest()
            ->take(10)
            ->get();
```